libs/csharp.json

```diff
@@ -65,5 +65,15 @@
   {"id": "cs.canopy", "imports": ["canopy"], "name": "canopy", "repo": "lefthandedgoat/canopy", "tags": ["web/testing"], "tech": ["testing"]},
   {"id": "cs.nunit", "imports": ["NUnit"], "name": "nunit", "repo": "nunit/nunit", "tags": ["unit-testing"], "tech": ["testing"]},
   {"id": "cs.xunit", "imports": ["Xunit"], "name": "xunit", "repo": "xunit/xunit", "tags": ["unit-testing"], "tech": ["testing"]},
+  {"id": "cs.zero-formatter", "imports": ["ZeroFormatter"], "name": "ZeroFormatter", "repo": "neuecc/ZeroFormatter", "tags": ["serialization"], "tech": ["efficient-data"]},
+  {"id": "cs.spreads-lmdb", "imports": ["Spreads.LMDB"], "name": "Spreads.LMDB", "repo": "Spreads/Spreads.LMDB", "tags": ["key-value"], "tech": ["database"]},
+  {"id": "cs.soundfingerprinting", "imports": ["SoundFingerprinting"], "name": "SoundFingerprinting", "repo": "AddictedCS/soundfingerprinting", "tags": ["audio-recognition"], "tech": ["audio"]},
+  {"id": "cs.fluent-assertions", "imports": ["FluentAssertions"], "name": "FluentAssertions", "repo": "fluentassertions/fluentassertions", "tags": ["assertions"], "tech": ["eng-tools"]},
+  {"id": "cs.cli-wrap", "imports": ["CliWrap"], "name": "CliWrap", "repo": "Tyrrrz/CliWrap", "tags": ["process-runner"], "tech": ["cli"]},
+  {"id": "cs.topshelf", "imports": ["Topshelf"], "name": "Topshelf", "repo": "Topshelf/Topshelf", "tags": ["windows-service"], "tech": ["app-blocks"]},
```

Comment:
maybe tech == app-arch? Topshelf describes itself as a framework, which means it defines what the application will look like. But since I am not familiar with Topshelf, I will leave this for you to decide.

Reply:
When i think about it Topshelf indeed forces architecture in a way. I'll change it to app-arch